src/bin/main.rs

```diff
@@ -12,6 +12,13 @@ use yarner::util::PathUtil;
 use yarner::{templates, MultipleTransclusionError, ProjectCreationError};
 
 fn main() {
+    match run() {
+        Ok(_) => {}
+        Err(err) => eprintln!("{}", err),
+    }
+}
```

Comment:
@adamreichold Any better idea how to solve the problem that escape sequences in error messages are ignored when just returning `Result<...>` from main? If I don't "wrap" it into a function that prints the error, I get things like \n or \" instead of newline or quotes.